src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -51,6 +52,13 @@
     @FXML
     private StackPane statusbarPlaceholder;
 
+    /**
+     * A wrapper around contents of DonnaFin view. Mainly for the global escape key function
+     * to direct users to home page.
+     */
+    @FXML
+    private StackPane viewWrapper;
```

Comment:
Do we need a viewWrapper for the entire window? I think the vBox here handles it.

![image](https://user-images.githubusercontent.com/64136922/136682847-9c972721-fcd7-46a2-8d45-a2a7f4ee05c2.png)